homeassistant/components/husqvarna_automower/sensor.py

```diff
@@ -26,6 +26,165 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+ERROR_KEY_LIST = [
```

Comment:
Is this list not available in the library to pull instead of listing explicitly here?

Reply:
Yes, it's available from the library. I got it from the library [here](https://github.com/home-assistant/core/pull/113165/commits/0ee6eab32758618b9cdffb869e4f6c484896329b). But I was asked [here](https://github.com/home-assistant/core/pull/113816#discussion_r1530778803) to literally list the options. So I expanded this to this PR. Should I change it back?

Reply:
No, let's leave it as it is. Should not break anything but for translations etc. I guess it's better to have it controlled.